kubernetes-model-generator/kubernetes-model-storageclass/src/main/resources/schema/kube-schema.json

```diff
@@ -1846,8 +2066,7 @@
       "additionalProperties": true,
       "javaType": "io.fabric8.kubernetes.api.model.storage.VolumeAttachment",
       "javaInterfaces": [
-        "io.fabric8.kubernetes.api.model.HasMetadata",
-        "io.fabric8.kubernetes.api.model.Namespaced"
```

Comment:
What changed?

Reply:
VolumeAttachment is a Cluster scoped resource:
```
kubernetes-client : $ kubectl api-resources | grep volumeattachment
volumeattachments                                      storage.k8s.io                         false        VolumeAttachment
kubernetes-client : $ 
```

Our test in UtilsTest has wrong assumption that it should be namespaced:
https://github.com/fabric8io/kubernetes-client/blob/64cc3b0153544fff2783dd485ef3fef592443b8d/kubernetes-client/src/test/java/io/fabric8/kubernetes/client/internal/UtilsTest.java#L281

Reply:
Yes, I saw that.
But why was the model generated like this before? has the scope changed in K8s? I haven't seen any changes specific to that in our go source code.

Reply:
Earlier we didn't had VolumeAttachment listed as a Cluster Scoped resource in pkg/schemagen:

https://github.com/fabric8io/kubernetes-client/blob/64cc3b0153544fff2783dd485ef3fef592443b8d/kubernetes-model-generator/pkg/schemagen/generate.go#L587-L590

In this PR, I realized it's generated as a namespaced resource and added it to [pkg/schemagen/generate.go](https://github.com/fabric8io/kubernetes-client/pull/2981/files#diff-beae899012aa1b496771283f5781ad3cfee88e185e4df8296e1068c7f7af15d4R623)

Reply:
Ok, this is the change I was missing:

https://github.com/fabric8io/kubernetes-client/blob/4b2aeea1e5d421ab1a7bc73bcd2e0c6621369417/kubernetes-model-generator/pkg/schemagen/generate.go#L622-L624